guide/english/bootstrap/forms/index.md

```diff
@@ -176,5 +176,15 @@ In combination with Bootstrap's predefined grid classes to align labels and grou
   </div>
 </form>
 ```
-    
-![Horizontal Form](https://github.com/TroyB12/Pictures/blob/master/Horizontal%20Form.PNG)
\ No newline at end of file
+
+![Horizontal Form](https://github.com/TroyB12/Pictures/blob/master/Horizontal%20Form.PNG)
```

Comment:
Please remove this picture, we do not accept these anymore since if the user removes them, these pictures disappear too